README.md

```diff
@@ -16,22 +17,24 @@ You can find more information on how to set up OpenELIS at our [docs page](http:
 
 #### Running docker compose With locally compiled/built Artifacts (ie the War file and React code) For Developers
 1. Clone the Repository
-
+       
          git clone https://github.com/I-TECH-UW/OpenELIS-Global-2.git 
+         NOTE: To  be able to build dataexport separatelyhttps://github.com/I-TECH-UW/dataexport and  ensure you are running on java 
 
 2. innitialize and build sub modules
 
-        git submodule update --init --recursive
         cd OpenELIS-Global-2/dataexport
+        git submodule update --init --recursive
         mvn clean install -DskipTests
 
 3.   Build the War file
 
-            cd OpenELIS-Global-2
+            Change directory(cd) .. since you are  already in OpenELIS-Global-2/dataexport
```

Comment:
Can we change this line from `Change directory(cd) .. since you are  already in OpenELIS-Global-2/dataexport` to just `cd ..` so that we just have simple commands in the instructions which everyone can copy.